app/components/Welcome.js

```diff
@@ -15,6 +16,17 @@ export default class Welcome extends Component {
       return (<Spinner />)
     }
 
+    // show welcome page only if the user did not hit
+    // home page ever
+    let electronStore = electron.remote.getGlobal('sharedObj').electronStore
```

Comment:
Here avoid using remote as possible as it does sync operations

Reply:
how can I make the electronStore instance available to all modules? this is the only way I found

Reply:
I believe it is better to use ipcRenderer right?